dashboard/test/controllers/home_controller_test.rb

```diff
@@ -366,7 +366,7 @@ class HomeControllerTest < ActionController::TestCase
   # TODO: remove this test when workshop_organizer is deprecated
   test 'workshop organizers see dashboard links' do
     sign_in create(:workshop_organizer, :with_terms_of_service)
-    query_count = 13
+    query_count = 14
```

Comment:
What's the extra query in all these tests- wondering if it's related to the unreliable query that popped up in 'facilitators see dashboard links'?

Reply:
yes, it is the same one: 
```
          Script Load [NO INDEX] (0.0ms)  SELECT `scripts`.* FROM `scripts`
        config/initializers/mysql_check_index_used.rb:22:in `execute'
        app/models/script.rb:319:in `all_scripts'
        app/models/script.rb:283:in `text_to_speech_unit_ids'
        app/controllers/home_controller.rb:157:in `init_homepage'
        app/controllers/home_controller.rb:65:in `home'
        app/helpers/read_replica_helper.rb:20:in `set_read_only_connection_for_block'
        test/controllers/home_controller_test.rb:372:in `block (2 levels) in <class:HomeControllerTest>'
        test/controllers/home_controller_test.rb:371:in `block in <class:HomeControllerTest>'
```
for whatever reason, this extra query only shows up in this test case when I run all of HomeControllerTest, not when I run this individual test case. again, it is puzzling why running more tests together could cause added queries in this test case. naively, I would expect that if more test cases are run together, then more data might be cached by active record by the time we get to this test case due to similar queries having been run already. however this seems to be the opposite.

Reply:
Ah, we can probably fix this by updating the default ignore filters that exclude queries related to the cached models: https://github.com/code-dot-org/code-dot-org/blob/a30e604da4598cbcd8b920bc83835c7f23d0b5cc/dashboard/test/testing/capture_queries.rb#L38-L47